README.md

```diff
@@ -9,4 +9,4 @@
 
 ## Building
 
-Use Visual Studio 2010+ to build this program. 
+Use Visual Studio 2017 or later to build this program.
```

Comment:
Why 2017?
What is wrong with 2013 or 2015?

Reply:
Converted back to 2010.

Reply:
Does this PR indeed compile fine (with your code changes) with VS 2010?

Reply:
🤔 I don't know, but it should. I have no VS 2010 to test.